cdap-client/src/main/java/co/cask/cdap/client/QueryClient.java

```diff
@@ -60,12 +57,13 @@ public String get() {
         if (config.getAccessToken() != null) {
           return config.getAccessToken().getValue();
         }
-
         return null;
       }
     };
 
-    exploreClient = new SuppliedAddressExploreClient(hostname, port, accessToken);
+    exploreClient = new SuppliedAddressExploreClient(hostname, port, accessToken,
+                                                     config.getConnectionConfig().isSSLEnabled(),
+                                                     config.isVerifySSLCert());
```

Comment:
I think the other two flags also need to come from the same place we get hostname, port and accessToken.